activestorage/app/models/active_storage/filename.rb

```diff
@@ -3,6 +3,8 @@
 # Encapsulates a string representing a filename to provide convenient access to parts of it and sanitization.
 # A Filename instance is returned by ActiveStorage::Blob#filename, and is comparable so it can be used for sorting.
 class ActiveStorage::Filename
+  require_dependency "active_storage/filename/parameters"
+
   include Comparable
```

Comment:
How about this one?

Reply:
That's https://ruby-doc.org/core-2.5.0/Comparable.html 🙂

Reply:
lol. True